pom.xml

```diff
@@ -11,14 +11,14 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>hal-explorer</artifactId>
-    <version>0.15.1-SNAPSHOT</version>
+    <version>0.16.0</version>
```

Comment:
```suggestion
    <version>0.16.0-SNAPSHOT</version>
```

Reply:
When I run `mvn release` it handles going from `SNAPSHOT` to release & tagging git.